core/system/inc/system.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2013-2015, ARM Limited, All Rights Reserved
```

Comment:
OK. I wasn't sure if it was a derivative work of the old file or a totally new thing.